### ISSUE TITLE
Update lib version

### ISSUE THREAD

**Issue by deckameron** [NetrisTV/ti.exoplayer#9]
Hi @drauggres,

I was looking for way to use 360 videos (surface_type : "spherical_view") and noticed that it is only possible in version [2.9.0](https://github.com/google/ExoPlayer/releases/tag/r2.9.0). It is now in [2.9.4](https://github.com/google/ExoPlayer/releases/tag/r2.9.4).

Could you please update it to a newer version? :-)
Thank you!

**Comment by drauggres** [comment]
Hi @deckameron.

I have tried to update to 2.9.0 in november, and was able to build module with it, but I couldn't get it work with Titanium because of [Java 8](https://github.com/google/ExoPlayer/blob/release-v2/RELEASENOTES.md#290): 
> Turn on Java 8 compiler support for the ExoPlayer library. Apps may need to add compileOptions { targetCompatibility JavaVersion.VERSION_1_8 } to their gradle settings to ensure bytecode compatibility.

[Ticket](https://jira.appcelerator.org/browse/AC-5688?focusedCommentId=436249&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-436249) in Appcelerators JIRA.

**Comment by drauggres** [comment]
If you want to try it yourself:
https://github.com/NetrisTV/ti.exoplayer/tree/test-2.9.1
```shell
git fetch origin
git checkout -b origin/test-2.9.1 test-2.9.1
cd android
ti clean
rm lib/*.aar lib/*.jar
cd ..
./gradlew app:tiBuild
```

**Comment by deckameron** [comment]
Thank you @drauggres ! I will give it a try.

**Comment by drauggres** [comment]
Sorry, I'm still on 7.4.1.
[Fix](https://github.com/NetrisTV/ti.exoplayer/commit/18ee374dc4b45c5b8be5a8530b3a7558b455585e) for 7.5.0 to the same branch.

BTW, It would be better if you just add new comments instead of editing (if this is not a typo or something like that). Github doesn't send notifications when comments are modified, only for new ones.

**Comment by drauggres** [comment]
https://jira.appcelerator.org/browse/AC-6174

**Comment by drauggres** [comment]
https://github.com/appcelerator/node-titanium-sdk/pull/80
https://github.com/appcelerator/titanium_mobile/pull/10809

**Comment by drauggres** [comment]
@deckameron 
You can try to build module from [test-2.9.6](https://github.com/NetrisTV/ti.exoplayer/tree/test-2.9.6) branch.
```javascript
const ExoPlayer = require("ru.netris.mobile.exoplayer");
const videoPlayer = ti_exoplayer.createVideoPlayer({
  ...
  surfaceType: ExoPlayer.SURFACE_TYPE_SPHERICAL_VIEW
});
```
If you don't have playservices in your app then comment out `extension-ima` in `app/build.gradle` or just delete `extension-ima-*.aar` from `lib` directory after module unpacking.

**Comment by deckameron** [comment]
Hi @drauggres 
I tried building the module but I got this error:

```javascript
[ERROR] /Users/sandrobarnabe/Documents/Modules/ti.exoplayer-test-2.9.6/android/build/intermediates/res/merged/layout/player_view_none.xml:2: error: No resource identifier found for attribute 'surface_type' in package 'ru.netris.mobile.exoplayer'

/Users/sandrobarnabe/Documents/Modules/ti.exoplayer-test-2.9.6/android/build/intermediates/res/merged/layout/player_view_spherical_view.xml:2: error: No resource identifier found for attribute 'surface_type' in package 'ru.netris.mobile.exoplayer'

/Users/sandrobarnabe/Documents/Modules/ti.exoplayer-test-2.9.6/android/build/intermediates/res/merged/layout/player_view_surface_view.xml:2: error: No resource identifier found for attribute 'surface_type' in package 'ru.netris.mobile.exoplayer'

/Users/sandrobarnabe/Documents/Modules/ti.exoplayer-test-2.9.6/android/build/intermediates/res/merged/layout/player_view_texture_view.xml:2: error: No resource identifier found for attribute 'surface_type' in package 'ru.netris.mobile.exoplayer'


[ERROR] Failed to generate R class
```

**Comment by drauggres** [comment]
Strange, just built it on different machine.
https://github.com/NetrisTV/ti.exoplayer/releases/tag/2.2.0-alpha

**Comment by drauggres** [comment]
https://jira.appcelerator.org/browse/TIMOB-27472

**Comment by drauggres** [comment]
Version [3.0.0](https://github.com/NetrisTV/ti.exoplayer/releases/tag/3.0.0) released with ExoPlayer v2.9.6
